docs/source/faq.rst

```diff
@@ -169,3 +169,24 @@ Next, run the integration tests as you would normally do, for example:
     mvn clean install -PIT
 
 Then connect to Tomcat using a remote debugger on port 8000, same as when normally debugging Tomcat.
+
+Why am I not seeing anything in "Admin Queues and Topics"?
+----------------------------------------------------------
+
+The most likely cause is a RMI connection error. By default RMI is using a random port.
+To set fixed port you have to edit ActiveMQ broker configuration, which is in the file ``/etc/activemq/instances-enabled/main/activemq.xml``.
+Add the following lines to broker configuration (name of currently using broker is in ``Admin/Settings/JMX/Broker name``):
+
+.. code-block:: xml
+
+	<managementContext>
+	    <managementContext createConnector="true" connectorPort="1099" rmiServerPort="1099" />
+	</managementContext>
+
+The next step is to restart ActiveMQ and MOTECH.
```

Comment:
Is it really necessary to restart MOTECH? Seems a little odd since we do not touch MOTECH at all in those steps.